### PR TITLE
New Getting Started Guide: Improvado Composable “Agentic” Data Platform with Snowflake

### PR DESCRIPTION
**Pull request by ilyasudakov**
Improvado Composable “Agentic” Data Platform with Snowflake - Getting Started Guide

Fork: https://github.com/ilyasudakov/sfquickstarts
Docs: https://improvado.io/help
I accept the [Snowflake's CLA](https://github.com/snowflakedb/CLA)

### CODE REVIEW

File: site/sfguides/src/improvado/improvado.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
+author:
```
Comment (iamontheinet) [inline site/sfguides/src/improvado/improvado.md]: Add author name--first and last

File: site/sfguides/src/improvado/improvado.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+By leveraging these capabilities, Improvado facilitates efficient data access and consumption, providing you with the tools necessary to extract valuable insights and maintain robust data governance practices.
+
+### Billing dashboard
+Improvado Billing Dashboard offers an in-depth analysis of your data usage, providing crucial segmented insights into extracted rows over your usage period. It also includes interactive features, enhancing your ability to examine and understand the data in a more dynamic and effective manner.
+
+![Billing dashboard](assets/billing_dashboard.png)
+
+
+<!-- ------------------------ -->
+## Conclusion
```
Comment (iamontheinet) [inline site/sfguides/src/improvado/improvado.md]: Rename this to "Conclusion And Resources"

**Review by iamontheinet — CHANGES_REQUESTED** [review]

File: site/sfguides/src/improvado/improvado.md (hunk 1 above)
Comment (ilyasudakov) [reply site/sfguides/src/improvado/improvado.md]: Added author name

File: site/sfguides/src/improvado/improvado.md (hunk 2 above)
Comment (ilyasudakov) [reply site/sfguides/src/improvado/improvado.md]: Fixed

**Pull request closed without merging**